spec/forms/hyrax/forms/resource_form_spec.rb

```diff
@@ -384,7 +384,13 @@
     end
 
     context 'when using a generic valkyrie adapter', valkyrie_adapter: :test_adapter do
+      before do
+        allow(Hyrax).to receive_message_chain(:config, :disable_wings).and_return(true) # rubocop:disable RSpec/MessageChain
+        hide_const("Wings") # disable_wings=true removes the Wings constant
+      end
```

Comment:
It already has a test using a non-wings adapter, but it doesn't fully simulate disabled wings.  When it is disabled, the `Wings` constant doesn't exist.  The new before block simulates `disable_wings` returning `true` and the `Wings` constant not existing.